src/utilities/views.py

```diff
@@ -13,3 +21,58 @@ def videos(request):
         form.save()
     context = {'videos': videos, 'heading': "Video Resources", 'form': form}
     return render(request, 'utilities/videos.html', context)
+
+
+def edit_post_request(request):
+    post = request.POST.copy()
```

Comment:
What is this method doing?  There's no docstring, and it seems hanging out here all weirdly, haha

Reply:
Ooooh!  I see what you're doing.  There are many ways of doing this, so for consistency, please see if you can override the `get_form_kwargs()` method instead of the `post()` method in the CreateView and UpdateView below.  For reference, see the `CourseAddStudent` method

Reply:
done